src/warnet/deploy.py

```diff
@@ -166,7 +166,7 @@ def run_plugins(directory, hook_value: HookValue, namespace, annex: Optional[dic
                 }
 
                 cmd = (
-                    f"{network_file_path.parent / entrypoint_path / Path('plugin.py')} entrypoint "
+                    f"python3 {network_file_path.parent / entrypoint_path / Path('plugin.py')} entrypoint "
```

Comment:
there's a trick to this, see how bitcoin test runner calls python from inside python:

https://github.com/bitcoin/bitcoin/blob/ee68b05f3d639b28b9d23b48324bc5f20ccfe49c/test/functional/test_runner.py#L783